Makefile

```diff
@@ -184,7 +184,19 @@ spell:
 
 man: man/avocado.1 man/avocado-rest-client.1
 
-.PHONY: source install clean check link
+show:
```

Comment:
Hmm, show what? I think better would be `make environment`, `make variables` or `make debug`.